app/editor/src/features/content/papers/Papers.tsx

```diff
@@ -68,7 +68,7 @@ const Papers: React.FC<IPapersProps> = (props) => {
   const [, setCurrentItems] = useLocalStorage('currentContent', {} as IContentSearchResult[]);
   const [currentItemId, setCurrentItemId] = useLocalStorage('currentContentItemId', -1);
 
-  const [, setIsLoading] = React.useState(false);
+  const [isLoading, setIsLoading] = React.useState(false);
```

Comment:
Instead of managing a separate state value you can `useApp` and check if there is an active request for content.

Reply:
I can do it in another PR.